xbmc/addons/kodi-addon-dev-kit/include/kodi/AddonBase.h

```diff
@@ -82,7 +82,9 @@ typedef enum ADDON_STATUS
   ///
   ADDON_STATUS_UNKNOWN,
 
+  /// @warning obsolete and no more used from Kodi!
   ///
+  /// Use kodi::SetSetting... during end to store changed settings
   ADDON_STATUS_NEED_SAVEDSETTINGS,
```

Comment:
Then I add a also a another commit where other functions becomes added and need a complete version change :D